### PR TITLE
Re-layout Raw Transaction Information

### PR DESCRIPTION
**Pull request by jubalm**
Updated the layout of raw transaction for data value to be closer to it's label and with more prominent colors. Pre-formatted data (raw) will also start using the new legible font

| Screenshot 2024-06-21 at 10 40 50 AM | Screenshot 2024-06-21 at 10 46 02 AM |
|:--:|:--:|
|_new_|_existing_|

### CODE REVIEW

File: app/css/interceptor.css [hunk 1]
```diff
@@ -1165,3 +1165,45 @@ summary:where(details[open] > *) {
 	text-overflow: ellipsis;
 	overflow: hidden;
 }
+
+.columnar {
+	--grid-cols: minmax(min-content,max-content) minmax(50%,1fr);
```
Comment (KillariDev) [inline app/css/interceptor.css:1170]:
```suggestion
	--grid-cols: minmax(min-content, max-content) minmax(50%, 1fr);
```

File: app/css/interceptor.css [hunk 2]
```diff
@@ -1165,3 +1165,45 @@ summary:where(details[open] > *) {
...
+	width: fit-content;
+	margin: 0 auto;
+}
+
+.columnar > * {
+	max-width: 100%;
+	min-width: 0;
+}
+
+.columnar dt {
```
Comment (KillariDev) [inline app/css/interceptor.css:1184]: can we come up with some better name for this?

File: app/css/interceptor.css [hunk 3]
```diff
@@ -1165,3 +1165,45 @@ summary:where(details[open] > *) {
...
+	justify-self: end;
+	text-align: right;
+	color: var(--subtitle-text-color);
+}
+
+.columnar dd {
```
Comment (KillariDev) [inline app/css/interceptor.css:1190]: can we come up with some better name for this?

**Comment by KillariDev** [comment]
Very good changes! Looks a lot better

File: app/css/interceptor.css (hunk 2 above)
Comment (jubalm) [reply app/css/interceptor.css]: how about `key-value-pair`?

**Review by KillariDev — APPROVED** [review]

**Pull request merged**